src/components/samples/SamplesListContent.js

```diff
@@ -25,13 +25,15 @@ import FiltersWarning from "../filters/FiltersWarning";
 import SamplesFilters from "./SamplesFilters";
 import mergedListQueryParams from "../../utils/mergedListQueryParams";
 
-const getTableColumns = (containersByID, individualsByID) => [
+const getTableColumns = (containersByID, individualsByID, sampleKinds) => [
     {
-      title: "Type",
-      dataIndex: "biospecimen_type",
+      title: "Sample Kind",
+      dataIndex: "sample_kind__name",
       sorter: true,
       width: 80,
-      render: (type) => <Tag>{type}</Tag>,
+      options: sampleKinds.map(x => ({ label: x.name, value: x.name })), // for getFilterProps
+      render: (_, sample) =>
+        <Tag>{sampleKinds.find(sk => sk.id === sample.sample_kind).name}</Tag>,
```

Comment:
Pass `sampleKindsByID` instead of finding by ID through an array.

Reply:
If you want you can pass `sampleKinds: state.sampleKinds` in `mapStateToProps` and use `sampleKinds.items` and `sampleKinds.itemsByID` here.